database_admin/update.sh

```diff
@@ -9,8 +9,16 @@ export PGSSLMODE=$DB_SSLMODE
 
 WAIT_FOR_EMPTY_DB=1 ./scripts/wait-for-services.sh
 
+# TODO: where is used this DB_INITIALIZED check result?
```

Comment:
Good to remove the DB_INITIALIZED, it was used to check whether to apply full schema or partial migration. We don;t need it anymore